_posts/2020-10-02-we-support-the-sustainable-development-goals.md

```diff
@@ -0,0 +1,60 @@
+
+By doing this, they are building effective partnerships (SDG 17.17) and laying the groundwork for OpenZaak to become a sustainable, resilient and inclusive (digital) infrastructure (SDG 9.1) for Dutch municipalities. During the market consultation, the OpenZaak community decided that public and private organizations should always steer codebase development together and wrote this into the codebase’s foundational documentation. Since the market consultation, municipalities and vendors have set-up a governance structure together to further develop and re-use OpenZaak. On a monthly basis they meet to manage the responsibility of developing and implementing the needs of the community - making for an effective and holistic partnership (SDG 17.17).
+
+### Goal 11: Sustainable Cities and Communities
+
+The Foundation for Public Code supports goal 11 by enabling sustainable collaboration on digital transformation at scale.
+
+Increasingly, urbanization is planned and managed using digital tools. Using open tools to do so, helps lay the groundwork for making urbanization participatory and inclusive. This builds trust with civil society and encourages further collaboration in the urbanization process.
+
+About a year ago we started working with [Signalen](https://publiccode.net/codebases/signalen.html), a system that enables residents to file a variety of nuisance reports, which are then automatically triaged and routed to the correct handling department within the municipality. The system has already been hugely successful in Amsterdam, and we are now helping a community of interested Dutch municipalities make Signalen reusable.
+
+Signalen's success can contribute to SDG 11.3: "By 2030, enhance inclusive and sustainable urbanization and capacity for participatory, integrated and sustainable human settlement planning and management in all countries."
+
+### Goal 16: Peace, Justice, and Strong Institutions
+
+The Foundation for Public Code contributes to achieving goal 16 by increasing transparency, quality public service delivery and participation.
+
+An important but often forgotten part of access to information is *how* policy is implemented. Developing public code projects in the open (publishing all code and accompanying documentation), allows the public to see how policy choices that are implemented through digital service provision work in practice. This helps the public understand how this affects their lives, for example algorithms shaped by public policy.
+
+A small practical example of this is work we've done with the [OpenZaak codebase](https://publiccode.net/codebases/openzaak.html). We are helping them move their conversations to open platforms - to GitHub when needed, or to a mailing list where anyone can join or read the archive. This makes it possible for anyone to see what is happening (SDG 16.6) and also participate if they have something to contribute (SDG 16.7).
```

Comment:
Can this be OpenZaak and Signalen for more equal airtime?

Reply:
@ElenaFdR yes absolutely. Good idea